src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -1313,8 +1319,8 @@ inlineToOpenXML' opts (Image attr@(imgident, _, _) alt (src, title)) = do
               , spPr
               ]
             ]
-        imgElt = mknode "w:r" [] $
-          mknode "w:drawing" [] $
+        imgElt = mknode "w:r" [] [ mknode "w:rPr" []
+                                   [ mknode "w:rStyle" [("w:val", fromStyleId inlineImageStyle)] ()], mknode "w:drawing" [] $
```

Comment:
The problem with this is that it always adds a w:rStyle, even when there isn't a style -- see the test failures.